src/posts/paper/13-q-and-a.ts

```diff
@@ -179,6 +179,54 @@ However, we have retracted this method for the following reasons:
 * Various studies, such as [Baker et al.](https://www.acpjournals.org/doi/10.7326/M20-7145) have shown that, for at least some pockets of healthcare workers, healthcare workers are no more likely to get infected through their work than the average person is to get infected through socializing.
 * The above Washington State report shows that people who work outside the home got COVID at about the average rate for the state.
 
+# Vaccines Q&A
+
+## How should we think about people who have been vaccinated?
+
+There are two questions that are important for understanding vaccines:
+
+1. How much safer for the vaccinated individual is exposure to COVID?
+2. How much less likely is it for a vaccinated individuals to expose others to COVID? (Or equivalently, how much less likely is a vaccinated individual to catch COVID?)
+
+We address these two questions individually.
+
+### 1. How much safer for the vaccinated individual is exposure to COVID?
+All currently approved vaccines are proven significantly reduce the chance of getting sick. If you do get sick after being vaccinated, you are significantly less likely to be hospitalized or die. However, one of the scary things about COVID is that cases with no outward symptoms have been found to cause severe lung damage [source?]. As a result:
+
+- For people who are trying to avoid COVID to avoid long term health problems, we do not recommend changing your budget (beyond what the reduction in microCOVIDs from #2 allows you to do).
```

Comment:
I've re-read this section like 3 times and I still don't understand. Is what you're saying: 
1. If you get vaccinated, we recommend not changing your budget. And you won't get any reduction in points. You only get a reduction in points if the person you are seeing is vaccinated themselves?
2. OR are you saying that we recommend not changing your budget. And all of your activities get a *0.2 multiplier to you (if Moderna or Pfizer).

Reply:
heavily edited.

All your activities get *0.2 multiplier to you.